play.pokemonshowdown.com/js/client-teambuilder.js

```diff
@@ -1789,34 +1792,71 @@
 					// in the future.
 					self.smogonSets[format] = false;
 				}
+				self.getUserSets(format);
 				self.importSetButtons();
 			}, 'text');
 		},
+		getUserSets: function (format) {
```

Comment:
Should perhaps place an upper limit on the number of sets per Pokémon? Just so someone doesn't have like 45 samples of a Pokémon in a given format, idk what would be a reasonable limit though